templates/project_root_containers/CMakeLists.txt

```diff
@@ -96,7 +96,21 @@ if(SIMPATH)
 endif()
 
 find_package2(PUBLIC ROOT  VERSION 6.10.00  REQUIRED)
+include(ROOTMacros)
+if(ROOT_vmc_FOUND)
+  set(VMCLIB ROOT::VMC)
+else()
+  find_package2(PUBLIC VMC REQUIRED)
+  set(VMCLIB VMCLibrary)
+endif()
+message(STATUS "Using VMC: ${VMCLIB}")
```

Comment:
```suggestion
```

Let's drop this or put it on different verbosity that is not printed by default.

Reply:
dropped.